src/js/worklets/chat/messages.js

```diff
@@ -0,0 +1,67 @@
+import { useDerivedValue, withTiming, interpolate } from 'react-native-reanimated';
+
+export function navigationHeaderOpacity(bigNameVisible) {
+  return useDerivedValue(function () {
+    'worklet';
+    return bigNameVisible.value ? withTiming(0) : withTiming(1);
+  });
```

Comment:
Could `bigNameVisible` be a ratom instead and avoid using the worklet, or is there a performance drawback to that?

Reply:
hi, thank you for reviewing PR. Yes they are drawbacks for using reagent atoms instead of worklets.
Please checkout [discord conversation ](https://discord.com/channels/1103692771585433630/1172058578153123840/1172058769627287622)

Reply:
big thumb-up on the points on ratoms and subs. It's become a nuisance to understand the data dependencies of a component cause everything is passed as maps of subs or ratoms to nested children and from there on it's the wild west.